tests/decorators/test_validation.py

```diff
@@ -1,7 +1,9 @@
 from unittest.mock import MagicMock
 
 import pytest
+from connexion.apis.flask_api import FlaskApi
 from connexion.decorators.validation import ParameterValidator
+from connexion.exceptions import ExtraParameterProblem
```

Comment:
Why is this import now needed? `ExtraParameterProblem` doesn't seem to be used in this file.

Reply:
Good catch. This is left over from a different implementation of the test I tried first.